src/primitives/index.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+/* eslint-disable import/no-unresolved */
+import reactPrimitives from 'react-primitives'
+
+import _StyledNativeComponent from '../models/StyledNativeComponent'
+import _constructWithOptions from '../constructors/constructWithOptions'
+
+import css from '../constructors/css'
+import ThemeProvider from '../models/ThemeProvider'
+import withTheme from '../hoc/withTheme'
+
+import type { Target } from '../types'
+
+const constructWithOptions = _constructWithOptions(css)
+const StyledNativeComponent = _StyledNativeComponent(
+  constructWithOptions,
+  reactPrimitives.StyleSheet,
+)
+const styled = (tag: Target) => constructWithOptions(StyledNativeComponent, tag)
+
+/* React native lazy-requires each of these modules for some reason, so let's
+*  assume it's for a good reason and not eagerly load them all */
+const aliases = 'Animated Image Text Touchable View '
```

Comment:
`Animated` isn't a target (assuming it's the same as in React Native), but it does have properties for it's target e.g. `Animated.View`, so this can't be added as an alias.